.gitattributes

```diff
@@ -0,0 +1,10 @@
+* text=auto
+
+*.py text diff=python
+*.java text diff=java
+*.gradle text diff=java
```

Comment:
I believe the `build.gradle` files are Groovy rather than regular Java, would `groovy` work here or is that not supported?

Reply:
Groovy is not supported, it is recommended to use Java.
![msedge_b7fLuB4cXh](https://github.com/UltraCosmetics/UltraCosmetics/assets/29678500/d7633922-7393-4754-9b3c-f3812b233678)